som/src/lvq_pak.lisp

```diff
@@ -53,7 +53,7 @@
      (current :accessor entries-current :initarg :current :initform 0
               :documentation "index of current data-entry inside data-entries")
      (entries :accessor entries-entries :initarg :entries :initform nil
-              :type #-ccl cons #+ccl list
+              :type #-ccl (or null cons) #+ccl list
```

Comment:
This can just be updated to be `:type list` since `(or null cons)` is === `list`

Reply:
Indeed, I'll update that.